jobbergateapi2/tests/test_pagination.py

```diff
@@ -19,17 +19,15 @@ def test_pagination_with_default_values():
     """
     pagination = Pagination()
 
-    assert pagination.q is None
     assert pagination.limit == 10
```

Comment:
Should default hardcoded values be defined at the top of the file or in a config area?

Reply:
Since pagination is a python class, it is defined in the class constructor